tool/site/client-react/src/components/prescription-summary/prescriptionLevelDetails.tsx

```diff
@@ -0,0 +1,135 @@
+import {SummaryList} from "nhsuk-react-components"
+import * as React from "react"
+import {CommunicationRequest, CommunicationRequestPayload, MedicationRequest} from "fhir/r4"
+import {formatDate} from "../../formatters/dates"
+import {
+  getNumberOfRepeatsAllowedExtension,
+  getNumberOfRepeatsIssuedExtension, getPerformerSiteTypeExtension,
+  getRepeatInformationExtension
+} from "../../fhir/customExtensions"
+
+export function createPrescriptionLevelDetails(medicationRequest: MedicationRequest, communicationRequests?: Array<CommunicationRequest>): PrescriptionLevelDetailsProps {
+  const prescriptionId = medicationRequest.groupIdentifier.value
+
+  const repeatExtension = getRepeatInformationExtension(medicationRequest.extension)
+  const repeatsIssuedExtension = getNumberOfRepeatsIssuedExtension(repeatExtension.extension)
+  const repeatIssued = repeatsIssuedExtension.valueInteger
+  const repeatsAllowedExtension = getNumberOfRepeatsAllowedExtension(repeatExtension.extension)
+  const repeatAllowed = repeatsAllowedExtension.valueInteger
+
+  const authoredOn = formatDate()
+  const startDate = medicationRequest.dispenseRequest.validityPeriod?.start ?? new Date().toISOString().slice(0, 10)
+  const nominatedOds = medicationRequest.dispenseRequest?.performer?.identifier?.value || ""
+
+  const nominatedTypeExtension = getPerformerSiteTypeExtension(medicationRequest.dispenseRequest.extension)
+  const nominatedTypeCode = nominatedTypeExtension.valueCoding.code
+  const nominatedType = getPharmacyTypeText(nominatedTypeCode)
+
+  const patientInstruction = communicationRequests
+    .flatMap(communicationRequest => communicationRequest.payload)
+    .filter(Boolean)
+    .filter(isContentStringPayload)
+    .map(payload => payload.contentString)
+    .join("\n")
```

Comment:
I think the `\n` will get lost when this is rendered. This might be a use case for our proposed "concatenate some strings with <br> in between" component (see patient and practitioner role addresses)